solr/packaging/build.gradle

```diff
@@ -160,3 +165,55 @@ distTar {
 distZip.enabled = false
 
 assemble.dependsOn installDist
+
+
+// We use Node to install BATS
+node {
+  // The Node.js project directory location
+  // This is where the package.json file and node_modules directory are located
+  // By default it is at the root of the current project
+  nodeProjectDir = project.ext.nodeProjectDir
+}
+
+task downloadBats(type: NpmTask) {
+  group = 'Build Dependency Download'
+  args = ["install", "bats",
+                     "ztombol/bats-support",
+                     "ztombol/bats-assert",
+  ]
+
+  inputs.files("${project.ext.nodeProjectDir}/package.json")
+  outputs.dir("${project.ext.nodeProjectDir}/node_modules/bats")
+  outputs.dir("${project.ext.nodeProjectDir}/node_modules/bats-support")
+  outputs.dir("${project.ext.nodeProjectDir}/node_modules/bats-assert")
+}
+
+task bats(type: Exec) {
+  dependsOn installDist
+  dependsOn downloadBats
+
+  def testOutput = "$buildDir/bats-output"
```

Comment:
Can we call this `integration-test-output`, and rename the task to `integrationTests` or something like that. To me, most people won't know what Bats is.